src/cmd-buildextend-installer

```diff
@@ -114,12 +127,19 @@ def generate_iso():
                 shutil.copyfileobj(fsrc, fdst)
             with open(tmp_cpio + '.gz', 'rb') as fsrc:
                 shutil.copyfileobj(fsrc, fdst)
+            # Pad for the ISO image.  We'll truncate it off again when
+            # copying the PXE initrd.
+            fdst.write(bytes(initrd_ignition_padding))
```

Comment:
might help here if the comment mentions `bytes()` is initialized with null bytes - `Pad for the ISO image initialized as null bytes` maybe?